pallets/handles/src/rpc/src/lib.rs

```diff
@@ -133,4 +137,15 @@ where
 		let result = api.validate_handle(at, base_handle);
 		map_rpc_result(result)
 	}
+
+	fn check_handle(&self, base_handle: String) -> RpcResult<CheckHandleResponse> {
+		let api = self.client.runtime_api();
+		let at = self.client.info().best_hash;
+		let base_handle: BaseHandle = base_handle
+			.into_bytes()
+			.try_into()
+			.map_err(|_| HandlesRpcError::InvalidHandle)?;
+		let result = api.check_handle(at, base_handle);
+		map_rpc_result(result)
+	}
```

Comment:
I'm not clear on the need to have both an RPC and a custom runtime call for this--also, per the latest guidance from Parity, aren't we supposed to *not* be creating custom RPCs any more?

Reply:
I did both as we have a lot of these and I wanted to continue the pattern for now.

I'm not sure what we want to do, but I'll add it to a conversation point for the Community Call: https://github.com/frequency-chain/frequency/issues/2240

Reply:
Given that a larger review/revamp of all RPCs seems to be around the corner anyway, I don't feel strongly either way, so non-blocking for me.